sunpy/util/config.py

```diff
@@ -78,54 +109,13 @@ def print_config():
 
 def _is_writable_dir(p):
     """Checks to see if a directory is writable"""
-    return os.path.isdir(p) and os.access(p, os.W_OK)
-
-
-def _get_home():
-    """Find user's home directory if possible.
-    Otherwise raise error.
-
-    """
-    path = os.path.expanduser("~")
-
-    if not os.path.isdir(path):
-        for evar in ('HOME', 'USERPROFILE', 'TMP'):
-            try:
-                path = os.environ[evar]
-                if os.path.isdir(path):
-                    break
-            except KeyError:
-                pass
-    if path:
-        return path
-    else:
-        raise RuntimeError('please define environment variable $HOME')
-
-
-def _find_config_files():
-    """Finds locations of SunPy configuration files"""
-    config_files = []
-    config_filename = 'sunpyrc'
-
-    # find default configuration file
-    module_dir = os.path.dirname(sunpy.__file__)
-    config_files.append(os.path.join(module_dir, 'data', 'sunpyrc'))
-
-    # if a user configuration file exists, add that to list of files to read
-    # so that any values set there will override ones specified in the default
-    # config file
-    config_path = _get_user_configdir()
-
-    if os.path.exists(os.path.join(config_path, config_filename)):
-        config_files.append(os.path.join(config_path, config_filename))
-
-    return config_files
+    return Path(p).is_dir() and os.access(p, os.W_OK)
 
 
 def _get_user_configdir():
     """
     Return the string representing the configuration dir.
-    The default is "HOME/.sunpy".  You can override this with the
+    The default is "HOME/.sunpy". You can override this with the
```

Comment:
I think it makes more sense if it is `$HOME`
```suggestion
    The default is "$HOME/.sunpy". You can override this with the
```